### ISSUE TITLE
Add Health Inspector enemy

### ISSUE THREAD

**Issue by richard-to** [richard-to/too-many-cooks#22]
- This needs more fleshing out
- Add a health inspector(s) sprite that will move around the map and try to disrupt the cooks
- I'm thinking the sprite will be some kind of rat person?

**Comment by elderferreiras** [comment]
If we choose to make the inspector a non-player, we could try to use logic similar to a zombie enemy: https://blog.ourcade.co/posts/2020/how-to-make-enemy-sprite-rotation-track-player-phaser-3/

**Comment by richard-to** [comment]
Yeah, I think we can experiment with a non-player health inspector. Should be interesting to implement.
